Lesson 4/yaroslav.partyckyi/docker_sailor.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+
+if [ "$EUID" -ne 0 ]
+then
+    echo ' . . . permission denied, please run as root'
+    exit
+fi
+
+echo -e "\n . . . remove/install docker\n"
+
+sudo apt-get update
+sudo apt-get remove docker docker-engine docker.io containerd runc
+sudo apt-get install apt-transport-https ca-certificates curl gnupg-agent software-properties-common
```

Comment:
After this operation, 81.5 MB of additional disk space will be used.
Do you want to continue? [Y/n]